File: generator/src/main/java/ch/bailu/gtk/tag/DocTag.java
package ch.bailu.gtk.tag;

public class DocTag extends Tag {
    private String text = "";

    public DocTag(Tag parent) {
        super(parent);
    }

    @Override
    public Tag getChild(String name, String prefix) {
        return new IgnoreTag(this);
    }

    @Override
    public void setText(String text) {
        this.text = text;
    }

    public String getText() {
        return text;
    }
}

File: generator/src/main/kotlin/ch/bailu/gtk/tag/DocTag.kt
package ch.bailu.gtk.tag

class DocTag(parent: TagWithParent): TagWithParent(parent) {
    var aText = ""

    override fun getChild(name: String, prefix: String): TagWithParent {
        return ignore()
    }

    override fun setText(text: String) {
        this.aText = text
    }

    fun getText(): String {
        return aText
    }

}